src/Components/SortControl/__tests__/SortControl.test.js

```diff
@@ -0,0 +1,31 @@
+import { render, screen } from '@testing-library/react';
+import { SortControl } from '../SortControl';
+import userEvent from '@testing-library/user-event';
+
+describe('SortControl', () => {
+  test('renders all options', () => {
+    render(<SortControl selectedOption={''} onSelect={jest.fn()} />);
+
+    expect(screen.getAllByRole('option')).toHaveLength(2);
```

Comment:
You can use constants in a test:

```suggestion
    expect(screen.getAllByRole('option')).toHaveLength(sortOptions.length);
```